### PR TITLE
qemu_v8: add mbedtls repo

### PR DESCRIPTION
**Pull request by vchong**
Signed-off-by: Victor Chong <victor.chong@linaro.org>

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by vchong** [comment]
Tag added. Ok for merge. Thanks!

File: qemu_v8.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
         <!-- Misc gits -->
         <project path="arm-trusted-firmware" name="ARM-software/arm-trusted-firmware.git" revision="34efb683e32254b8c325ac3071c5776d243a7b99" />
+        <project path="mbedtls" name="ARMmbed/mbedtls.git" revision="refs/tags/mbedtls-2.16.0" />
```
Comment (jforissier) [inline qemu_v8.xml]: `clone-depth=1`?

**Comment by jforissier** [comment]
While you're at it, you could say in the commit message why you are adding this.

`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by vchong** [comment]
> While you're at it, you could say in the commit message why you are adding this.

Good idea thanks! Done.

File: qemu_v8.xml (hunk 1 above)
Comment (jforissier) [reply qemu_v8.xml]: ?

Comment (jforissier) [reply qemu_v8.xml]: So, no `clone-depth="1"`?

Comment (jbech-linaro) [reply qemu_v8.xml]: @vchong , please add the `clone-depth="1"` as mentioned by @jforissier  (for more info see [our docs](https://optee.readthedocs.io/building/gits/manifest.html#when-to-use-clone-depth-1)).

**Comment by jbech-linaro** [comment]
With the clone-depth resolved, then:
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by vchong** [comment]
Also fixed the alignment and ordering. All tags added. Ok to merged? Thanks!

**Comment by jforissier** [comment]
@vchong no worries, thanks.

**Pull request merged**